File: problems/70/solution.java

class Solution {
	public String convertToTitle(int a) {
		String result = "";
		while (a > 0) {
			int r = (a - 1) % 26;
			result = (char) ('A' + r) + result;
			a = (a - 1) / 26;
		}
		return result;
	}
}

File: problems/70/solution.py

class Solution:
    # @param A : integer
    # @return a strings
    def convertToTitle(self, a):
        result = ""
        while (a > 0):
            r = (a - 1) % 26
            result = chr(ord('A') + r) + result
            a = (a - 1) / 26
        return result


